02-molecules/cta/Cta.component.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '../../01-atoms/buttons/Button.component';
+
+const withModifiers = (className) => (modifiers) =>
+  `${className} ${modifiers.map((mod) => ` ${className}--${mod}`)}`;
+
+const Cta = ({
+  cta_modifiers = [],
+  heading,
+  button_modifiers,
+  button_text,
+}) => (
+  <div className={withModifiers('cta')(cta_modifiers)}>
+    <h2 className="cta__heading">{heading}</h2>
```

Comment:
I know that this storybook project is meant to mirror other parts of the design system, so my comment here may make no sense...

Should there be a heading atom? This doesn't feel very molecuely having only a single atom.